package.json

```diff
@@ -65,7 +65,7 @@
     "rimraf": "^3.0.2",
     "ts-node": "^9.1.1",
     "tslint": "^5.20.1",
-    "typedoc": "^0.23.8",
+    "typedoc": "^0.19.2",
```

Comment:
looks unrelated change

Reply:
I did it for documentation warning, not a breaking change or anything